src/interactions/modals/embedModal.ts

```diff
@@ -0,0 +1,30 @@
+import { Modal } from "../interactionClasses";
+import { EmbedBuilder, ModalSubmitInteraction } from "discord.js";
+import { addDefaultEmbedFooter } from "../../util/misc/embeds";
+import { config } from "../../config";
+
+class EmbedModal extends Modal {
+	constructor() {
+		super("embed");
+	}
+
+	async execute(interaction: ModalSubmitInteraction, customId: string[]): Promise<void> {
+		const showAuthor = customId.shift() === "true";
+		const title = interaction.fields.getTextInputValue("title");
+		const description = interaction.fields.getTextInputValue("description");
+
+		let embed = new EmbedBuilder().setTitle(title).setDescription(description);
+
```

Comment:
Should probably check that everything in total is less than 6000 characters.
Reference the [Discord.js Guide](https://discordjs.guide/popular-topics/embeds.html#embed-limits) for embed limits.